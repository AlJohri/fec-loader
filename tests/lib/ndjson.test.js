const { ObjectReadableMock, ObjectWritableMock } = require('stream-mock');

const ndjson = require('../../lib/ndjson');
const fs = require('fs');

test('ndjson', async done => {
    const input = fs.createReadStream(__dirname + '/../fixtures/989514.fec');
    const writer = new ObjectWritableMock();

    ndjson({
        in: input,
        out: writer
    });

    writer.on('finish', () => {
        expect(writer.data).toMatchSnapshot();
        done();
    });
});
