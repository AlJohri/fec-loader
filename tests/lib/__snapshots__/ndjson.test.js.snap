// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ndjson 1`] = `
Array [
  "{\\"record_type\\":\\"HDR\\",\\"ef_type\\":\\"FEC\\",\\"fec_version\\":\\"8.1\\",\\"soft_name\\":\\"CMDI CRIMSON FILER\\",\\"soft_ver\\":\\"2.0\\",\\"report_id\\":null,\\"report_number\\":null,\\"comment\\":null}
",
  "{\\"form_type\\":\\"F3PN\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"committee_name\\":\\"ROMNEY FOR PRESIDENT, INC.\\",\\"change_of_address\\":null,\\"street_1\\":\\"C/O RED CURVE SOLUTIONS, LLC\\",\\"street_2\\":\\"500 CUMMINGS CENTER, SUITE 4400\\",\\"city\\":\\"BEVERLY\\",\\"state\\":\\"MA\\",\\"zip_code\\":\\"01915\\",\\"activity_primary\\":null,\\"activity_general\\":null,\\"report_code\\":\\"YE\\",\\"election_code\\":null,\\"date_of_election\\":null,\\"state_of_election\\":null,\\"coverage_from_date\\":\\"20141125\\",\\"coverage_through_date\\":\\"20141231\\",\\"treasurer_last_name\\":\\"Crate\\",\\"treasurer_first_name\\":\\"Darrell\\",\\"treasurer_middle_name\\":null,\\"treasurer_prefix\\":null,\\"treasurer_suffix\\":null,\\"date_signed\\":\\"20150130\\",\\"col_a_cash_on_hand_beginning_period\\":\\"2176183.62\\",\\"col_a_total_receipts\\":\\"12592.00\\",\\"col_a_subtotal\\":\\"2188775.62\\",\\"col_a_total_disbursements\\":\\"157829.70\\",\\"col_a_cash_on_hand_close_of_period\\":\\"2030945.92\\",\\"col_a_debts_to\\":\\"0.00\\",\\"col_a_debts_by\\":\\"0.00\\",\\"col_a_expenditures_subject_to_limits\\":\\"0.00\\",\\"col_a_net_contributions\\":\\"-617988.26\\",\\"col_a_net_operating_expenditures\\":\\"16330140.35\\",\\"col_a_federal_funds\\":\\"0.00\\",\\"col_a_individuals_itemized\\":\\"0.00\\",\\"col_a_individuals_unitemized\\":\\"0.00\\",\\"col_a_individual_contribution_total\\":\\"0.00\\",\\"col_a_political_party_committees_receipts\\":\\"0.00\\",\\"col_a_other_political_committees_pacs\\":\\"0.00\\",\\"col_a_the_candidate\\":\\"0.00\\",\\"col_a_total_contributions\\":\\"0.00\\",\\"col_a_transfers_from_aff_other_party_cmttees\\":\\"0.00\\",\\"col_a_received_from_or_guaranteed_by_cand\\":\\"0.00\\",\\"col_a_other_loans\\":\\"0.00\\",\\"col_a_total_loans\\":\\"0.00\\",\\"col_a_operating\\":\\"0.00\\",\\"col_a_fundraising\\":\\"0.00\\",\\"col_a_legal_and_accounting\\":\\"0.00\\",\\"col_a_total_offsets_to_expenditures\\":\\"0.00\\",\\"col_a_other_receipts\\":\\"12592.00\\",\\"col_a_operating_expenditures\\":\\"157829.70\\",\\"col_a_transfers_to_other_authorized_committees\\":\\"0.00\\",\\"col_a_fundraising_disbursements\\":\\"0.00\\",\\"col_a_exempt_legal_accounting_disbursement\\":\\"0.00\\",\\"col_a_made_or_guaranteed_by_candidate\\":\\"0.00\\",\\"col_a_other_repayments\\":\\"0.00\\",\\"col_a_total_loan_repayments_made\\":\\"0.00\\",\\"col_a_individuals\\":\\"0.00\\",\\"col_a_political_party_committees_refunds\\":\\"0.00\\",\\"col_a_other_political_committees\\":\\"0.00\\",\\"col_a_total_contributions_refunds\\":\\"0.00\\",\\"col_a_other_disbursements\\":\\"0.00\\",\\"col_a_items_on_hand_to_be_liquidated\\":\\"0.00\\",\\"col_a_alabama\\":\\"0.00\\",\\"col_a_alaska\\":\\"0.00\\",\\"col_a_arizona\\":\\"0.00\\",\\"col_a_arkansas\\":\\"0.00\\",\\"col_a_california\\":\\"0.00\\",\\"col_a_colorado\\":\\"0.00\\",\\"col_a_connecticut\\":\\"0.00\\",\\"col_a_delaware\\":\\"0.00\\",\\"col_a_dist_of_columbia\\":\\"0.00\\",\\"col_a_florida\\":\\"0.00\\",\\"col_a_georgia\\":\\"0.00\\",\\"col_a_hawaii\\":\\"0.00\\",\\"col_a_idaho\\":\\"0.00\\",\\"col_a_illinois\\":\\"0.00\\",\\"col_a_indiana\\":\\"0.00\\",\\"col_a_iowa\\":\\"0.00\\",\\"col_a_kansas\\":\\"0.00\\",\\"col_a_kentucky\\":\\"0.00\\",\\"col_a_louisiana\\":\\"0.00\\",\\"col_a_maine\\":\\"0.00\\",\\"col_a_maryland\\":\\"0.00\\",\\"col_a_massachusetts\\":\\"0.00\\",\\"col_a_michigan\\":\\"0.00\\",\\"col_a_minnesota\\":\\"0.00\\",\\"col_a_mississippi\\":\\"0.00\\",\\"col_a_missouri\\":\\"0.00\\",\\"col_a_montana\\":\\"0.00\\",\\"col_a_nebraska\\":\\"0.00\\",\\"col_a_nevada\\":\\"0.00\\",\\"col_a_new_hampshire\\":\\"0.00\\",\\"col_a_new_jersey\\":\\"0.00\\",\\"col_a_new_mexico\\":\\"0.00\\",\\"col_a_new_york\\":\\"0.00\\",\\"col_a_north_carolina\\":\\"0.00\\",\\"col_a_north_dakota\\":\\"0.00\\",\\"col_a_ohio\\":\\"0.00\\",\\"col_a_oklahoma\\":\\"0.00\\",\\"col_a_oregon\\":\\"0.00\\",\\"col_a_pennsylvania\\":\\"0.00\\",\\"col_a_rhode_island\\":\\"0.00\\",\\"col_a_south_carolina\\":\\"0.00\\",\\"col_a_south_dakota\\":\\"0.00\\",\\"col_a_tennessee\\":\\"0.00\\",\\"col_a_texas\\":\\"0.00\\",\\"col_a_utah\\":\\"0.00\\",\\"col_a_vermont\\":\\"0.00\\",\\"col_a_virginia\\":\\"0.00\\",\\"col_a_washington\\":\\"0.00\\",\\"col_a_west_virginia\\":\\"0.00\\",\\"col_a_wisconsin\\":\\"0.00\\",\\"col_a_wyoming\\":\\"0.00\\",\\"col_a_puerto_rico\\":\\"0.00\\",\\"col_a_guam\\":\\"0.00\\",\\"col_a_virgin_islands\\":\\"0.00\\",\\"col_a_totals\\":\\"0.00\\",\\"col_b_federal_funds\\":\\"0.00\\",\\"col_b_individuals_itemized\\":\\"440290.98\\",\\"col_b_individuals_unitemized\\":\\"214971.78\\",\\"col_b_individual_contribution_total\\":\\"655262.76\\",\\"col_b_political_party_committees_receipts\\":\\"0.00\\",\\"col_b_other_political_committees_pacs\\":\\"38713.49\\",\\"col_b_the_candidate\\":\\"0.00\\",\\"col_b_total_contributions_other_than_loans\\":\\"693976.25\\",\\"col_b_transfers_from_aff_other_party_cmttees\\":\\"3490846.12\\",\\"col_b_received_from_or_guaranteed_by_cand\\":\\"0.00\\",\\"col_b_other_loans\\":\\"0.00\\",\\"col_b_total_loans\\":\\"0.00\\",\\"col_b_operating\\":\\"7285341.61\\",\\"col_b_fundraising\\":\\"0.00\\",\\"col_b_legal_and_accounting\\":\\"0.00\\",\\"col_b_total_offsets_to_operating_expenditures\\":\\"7285341.61\\",\\"col_b_other_receipts\\":\\"3598579.11\\",\\"col_b_total_receipts\\":\\"15068743.09\\",\\"col_b_operating_expenditures\\":\\"23615481.96\\",\\"col_b_transfers_to_other_authorized_committees\\":\\"444686.59\\",\\"col_b_fundraising_disbursements\\":\\"0.00\\",\\"col_b_exempt_legal_accounting_disbursement\\":\\"0.00\\",\\"col_b_made_or_guaranteed_by_the_candidate\\":\\"0.00\\",\\"col_b_other_repayments\\":\\"1500000.00\\",\\"col_b_total_loan_repayments_made\\":\\"1500000.00\\",\\"col_b_individuals\\":\\"1281800.02\\",\\"col_b_political_party_committees_refunds\\":\\"0.00\\",\\"col_b_other_political_committees\\":\\"30164.49\\",\\"col_b_total_contributions_refunds\\":\\"1311964.51\\",\\"col_b_other_disbursements\\":\\"184865.62\\",\\"col_b_total_disbursements\\":\\"27056998.68\\",\\"col_b_alabama\\":\\"0.00\\",\\"col_b_alaska\\":\\"0.00\\",\\"col_b_arizona\\":\\"0.00\\",\\"col_b_arkansas\\":\\"0.00\\",\\"col_b_california\\":\\"0.00\\",\\"col_b_colorado\\":\\"0.00\\",\\"col_b_connecticut\\":\\"0.00\\",\\"col_b_delaware\\":\\"0.00\\",\\"col_b_dist_of_columbia\\":\\"0.00\\",\\"col_b_florida\\":\\"0.00\\",\\"col_b_georgia\\":\\"0.00\\",\\"col_b_hawaii\\":\\"0.00\\",\\"col_b_idaho\\":\\"0.00\\",\\"col_b_illinois\\":\\"0.00\\",\\"col_b_indiana\\":\\"0.00\\",\\"col_b_iowa\\":\\"0.00\\",\\"col_b_kansas\\":\\"0.00\\",\\"col_b_kentucky\\":\\"0.00\\",\\"col_b_louisiana\\":\\"0.00\\",\\"col_b_maine\\":\\"0.00\\",\\"col_b_maryland\\":\\"0.00\\",\\"col_b_massachusetts\\":\\"0.00\\",\\"col_b_michigan\\":\\"0.00\\",\\"col_b_minnesota\\":\\"0.00\\",\\"col_b_mississippi\\":\\"0.00\\",\\"col_b_missouri\\":\\"0.00\\",\\"col_b_montana\\":\\"0.00\\",\\"col_b_nebraska\\":\\"0.00\\",\\"col_b_nevada\\":\\"0.00\\",\\"col_b_new_hampshire\\":\\"0.00\\",\\"col_b_new_jersey\\":\\"0.00\\",\\"col_b_new_mexico\\":\\"0.00\\",\\"col_b_new_york\\":\\"0.00\\",\\"col_b_north_carolina\\":\\"0.00\\",\\"col_b_north_dakota\\":\\"0.00\\",\\"col_b_ohio\\":\\"0.00\\",\\"col_b_oklahoma\\":\\"0.00\\",\\"col_b_oregon\\":\\"0.00\\",\\"col_b_pennsylvania\\":\\"0.00\\",\\"col_b_rhode_island\\":\\"0.00\\",\\"col_b_south_carolina\\":\\"0.00\\",\\"col_b_south_dakota\\":\\"0.00\\",\\"col_b_tennessee\\":\\"0.00\\",\\"col_b_texas\\":\\"0.00\\",\\"col_b_utah\\":\\"0.00\\",\\"col_b_vermont\\":\\"0.00\\",\\"col_b_virginia\\":\\"0.00\\",\\"col_b_washington\\":\\"0.00\\",\\"col_b_west_virginia\\":\\"0.00\\",\\"col_b_wisconsin\\":\\"0.00\\",\\"col_b_wyoming\\":\\"0.00\\",\\"col_b_puerto_rico\\":\\"0.00\\",\\"col_b_guam\\":\\"0.00\\",\\"col_b_virgin_islands\\":\\"0.00\\",\\"col_b_totals\\":\\"0.00\\"}
",
  "{\\"form_type\\":\\"SA21\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id\\":\\"SA21.001\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"contributor_organization_name\\":\\"FLS CONNECT, LLC\\",\\"contributor_last_name\\":\\"\\",\\"contributor_first_name\\":\\"\\",\\"contributor_middle_name\\":\\"\\",\\"contributor_prefix\\":\\"\\",\\"contributor_suffix\\":\\"\\",\\"contributor_street_1\\":\\"7300 HUDSON BLVD\\",\\"contributor_street_2\\":\\"STE 270\\",\\"contributor_city\\":\\"SAINT PAUL\\",\\"contributor_state\\":\\"MN\\",\\"contributor_zip_code\\":\\"55128\\",\\"election_code\\":\\"G2012\\",\\"election_other_description\\":null,\\"contribution_date\\":\\"20141203\\",\\"contribution_amount\\":\\"1640.25\\",\\"contribution_aggregate\\":\\"40472.19\\",\\"contribution_purpose_descrip\\":\\"LIST RENTAL\\",\\"contributor_employer\\":\\"\\",\\"contributor_occupation\\":\\"\\",\\"donor_committee_fec_id\\":null,\\"donor_committee_name\\":\\"\\",\\"donor_candidate_fec_id\\":null,\\"donor_candidate_last_name\\":\\"\\",\\"donor_candidate_first_name\\":\\"\\",\\"donor_candidate_middle_name\\":\\"\\",\\"donor_candidate_prefix\\":\\"\\",\\"donor_candidate_suffix\\":\\"\\",\\"donor_candidate_office\\":\\"\\",\\"donor_candidate_state\\":null,\\"donor_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street1\\":\\"\\",\\"conduit_street2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"FAIR MARKET VALUE\\",\\"reference_code\\":null}
",
  "{\\"form_type\\":\\"SA21\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id\\":\\"SA21.002\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"contributor_organization_name\\":\\"FLS CONNECT, LLC\\",\\"contributor_last_name\\":\\"\\",\\"contributor_first_name\\":\\"\\",\\"contributor_middle_name\\":\\"\\",\\"contributor_prefix\\":\\"\\",\\"contributor_suffix\\":\\"\\",\\"contributor_street_1\\":\\"7300 HUDSON BLVD\\",\\"contributor_street_2\\":\\"STE 270\\",\\"contributor_city\\":\\"SAINT PAUL\\",\\"contributor_state\\":\\"MN\\",\\"contributor_zip_code\\":\\"55128\\",\\"election_code\\":\\"G2012\\",\\"election_other_description\\":null,\\"contribution_date\\":\\"20141203\\",\\"contribution_amount\\":\\"3667.75\\",\\"contribution_aggregate\\":\\"40472.19\\",\\"contribution_purpose_descrip\\":\\"LIST RENTAL\\",\\"contributor_employer\\":\\"\\",\\"contributor_occupation\\":\\"\\",\\"donor_committee_fec_id\\":null,\\"donor_committee_name\\":\\"\\",\\"donor_candidate_fec_id\\":null,\\"donor_candidate_last_name\\":\\"\\",\\"donor_candidate_first_name\\":\\"\\",\\"donor_candidate_middle_name\\":\\"\\",\\"donor_candidate_prefix\\":\\"\\",\\"donor_candidate_suffix\\":\\"\\",\\"donor_candidate_office\\":\\"\\",\\"donor_candidate_state\\":null,\\"donor_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street1\\":\\"\\",\\"conduit_street2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"FAIR MARKET VALUE\\",\\"reference_code\\":null}
",
  "{\\"form_type\\":\\"SA21\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id\\":\\"SA21.003\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"contributor_organization_name\\":\\"NEWSMAX MEDIA\\",\\"contributor_last_name\\":\\"\\",\\"contributor_first_name\\":\\"\\",\\"contributor_middle_name\\":\\"\\",\\"contributor_prefix\\":\\"\\",\\"contributor_suffix\\":\\"\\",\\"contributor_street_1\\":\\"560 VILLAGE BLVD.\\",\\"contributor_street_2\\":\\"STE 120\\",\\"contributor_city\\":\\"WEST PALM BEACH\\",\\"contributor_state\\":\\"FL\\",\\"contributor_zip_code\\":\\"33409\\",\\"election_code\\":\\"G2012\\",\\"election_other_description\\":null,\\"contribution_date\\":\\"20141203\\",\\"contribution_amount\\":\\"7284.00\\",\\"contribution_aggregate\\":\\"68171.27\\",\\"contribution_purpose_descrip\\":\\"LIST RENTAL\\",\\"contributor_employer\\":\\"\\",\\"contributor_occupation\\":\\"\\",\\"donor_committee_fec_id\\":null,\\"donor_committee_name\\":\\"\\",\\"donor_candidate_fec_id\\":null,\\"donor_candidate_last_name\\":\\"\\",\\"donor_candidate_first_name\\":\\"\\",\\"donor_candidate_middle_name\\":\\"\\",\\"donor_candidate_prefix\\":\\"\\",\\"donor_candidate_suffix\\":\\"\\",\\"donor_candidate_office\\":\\"\\",\\"donor_candidate_state\\":null,\\"donor_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street1\\":\\"\\",\\"conduit_street2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"FAIR MARKET VALUE\\",\\"reference_code\\":null}
",
  "{\\"form_type\\":\\"SB23\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id_number\\":\\"SB23.013\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"payee_organization_name\\":\\"AIR CHARTER\\",\\"payee_last_name\\":\\"\\",\\"payee_first_name\\":\\"\\",\\"payee_middle_name\\":\\"\\",\\"payee_prefix\\":\\"\\",\\"payee_suffix\\":\\"\\",\\"payee_street_1\\":\\"4151 N. MULBERRY DR.\\",\\"payee_street_2\\":\\"SUITE 250\\",\\"payee_city\\":\\"KANSAS CITY\\",\\"payee_state\\":\\"MO\\",\\"payee_zip_code\\":\\"64116\\",\\"election_code\\":null,\\"election_other_description\\":null,\\"expenditure_date\\":\\"20141210\\",\\"expenditure_amount\\":\\"5740.80\\",\\"semi_annual_refunded_bundled_amt\\":null,\\"expenditure_purpose_descrip\\":\\"TRAVEL: AIR\\",\\"category_code\\":null,\\"beneficiary_committee_fec_id\\":null,\\"beneficiary_committee_name\\":null,\\"beneficiary_candidate_fec_id\\":null,\\"beneficiary_candidate_last_name\\":\\"\\",\\"beneficiary_candidate_first_name\\":\\"\\",\\"beneficiary_candidate_middle_name\\":\\"\\",\\"beneficiary_candidate_prefix\\":\\"\\",\\"beneficiary_candidate_suffix\\":\\"\\",\\"beneficiary_candidate_office\\":null,\\"beneficiary_candidate_state\\":null,\\"beneficiary_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street_1\\":\\"\\",\\"conduit_street_2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"IN-KIND CONTRIBUTION: SASSE FOR US SENATE; SEE POST-GEN TRAN ID SB29.05\\",\\"reference_to_si_or_sl_system_code_that_identifies_the_account\\":null}
",
  "{\\"form_type\\":\\"SB23\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id_number\\":\\"SB23.004\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"payee_organization_name\\":\\"ATCHLEY & ASSOCIATES\\",\\"payee_last_name\\":\\"\\",\\"payee_first_name\\":\\"\\",\\"payee_middle_name\\":\\"\\",\\"payee_prefix\\":\\"\\",\\"payee_suffix\\":\\"\\",\\"payee_street_1\\":\\"6850 AUSTIN CENTER BOULEVARD\\",\\"payee_street_2\\":\\"SUITE 180\\",\\"payee_city\\":\\"AUSTIN\\",\\"payee_state\\":\\"TX\\",\\"payee_zip_code\\":\\"78731\\",\\"election_code\\":null,\\"election_other_description\\":null,\\"expenditure_date\\":\\"20141222\\",\\"expenditure_amount\\":\\"299.00\\",\\"semi_annual_refunded_bundled_amt\\":null,\\"expenditure_purpose_descrip\\":\\"ACCOUNTING SERVICES\\",\\"category_code\\":null,\\"beneficiary_committee_fec_id\\":null,\\"beneficiary_committee_name\\":null,\\"beneficiary_candidate_fec_id\\":null,\\"beneficiary_candidate_last_name\\":\\"\\",\\"beneficiary_candidate_first_name\\":\\"\\",\\"beneficiary_candidate_middle_name\\":\\"\\",\\"beneficiary_candidate_prefix\\":\\"\\",\\"beneficiary_candidate_suffix\\":\\"\\",\\"beneficiary_candidate_office\\":null,\\"beneficiary_candidate_state\\":null,\\"beneficiary_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street_1\\":\\"\\",\\"conduit_street_2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"\\",\\"reference_to_si_or_sl_system_code_that_identifies_the_account\\":null}
",
  "{\\"form_type\\":\\"SB23\\",\\"filer_committee_id_number\\":\\"C00431171\\",\\"transaction_id_number\\":\\"SB23.005\\",\\"back_reference_tran_id_number\\":null,\\"back_reference_sched_name\\":null,\\"entity_type\\":\\"ORG\\",\\"payee_organization_name\\":\\"CMDI\\",\\"payee_last_name\\":\\"\\",\\"payee_first_name\\":\\"\\",\\"payee_middle_name\\":\\"\\",\\"payee_prefix\\":\\"\\",\\"payee_suffix\\":\\"\\",\\"payee_street_1\\":\\"1593 SPRING HILL ROAD\\",\\"payee_street_2\\":\\"SUITE 450\\",\\"payee_city\\":\\"TYSONS CORNER\\",\\"payee_state\\":\\"VA\\",\\"payee_zip_code\\":\\"22182\\",\\"election_code\\":null,\\"election_other_description\\":null,\\"expenditure_date\\":\\"20141222\\",\\"expenditure_amount\\":\\"27500.00\\",\\"semi_annual_refunded_bundled_amt\\":null,\\"expenditure_purpose_descrip\\":\\"DATA MANAGEMENT SERVICES\\",\\"category_code\\":null,\\"beneficiary_committee_fec_id\\":null,\\"beneficiary_committee_name\\":null,\\"beneficiary_candidate_fec_id\\":null,\\"beneficiary_candidate_last_name\\":\\"\\",\\"beneficiary_candidate_first_name\\":\\"\\",\\"beneficiary_candidate_middle_name\\":\\"\\",\\"beneficiary_candidate_prefix\\":\\"\\",\\"beneficiary_candidate_suffix\\":\\"\\",\\"beneficiary_candidate_office\\":null,\\"beneficiary_candidate_state\\":null,\\"beneficiary_candidate_district\\":null,\\"conduit_name\\":\\"  \\",\\"conduit_street_1\\":\\"\\",\\"conduit_street_2\\":\\"\\",\\"conduit_city\\":\\"\\",\\"conduit_state\\":null,\\"conduit_zip_code\\":null,\\"memo_code\\":null,\\"memo_text_description\\":\\"\\",\\"reference_to_si_or_sl_system_code_that_identifies_the_account\\":null}
",
]
`;
